src/components/sdk/DeviceSelection/DeviceInput.tsx

```diff
@@ -12,47 +18,58 @@ interface Props extends BaseSdkProps {
   label: string;
   notFoundMsg: string;
   devices: DeviceType[];
-  selectedDeviceId: SelectedDeviceId;
+  selectedDevice:
```

Comment:
Can this be `AudioInputDevice` | `VideoInputDevice` | `undefined` ? Is there a reason for separating the types here?

Reply:
The type is exactly same. The only reason I did not do that is because it is confusing.

`selectedDevice` also applies for `AudioOutputDevice`. If `selectedDevice` is type of `AudioInputDevice | VideoInputDevice | undefined`, builder might be curious where is `AudioOutputDevice`. And they have to check the JS SDK code and figure out that there's no type of `AudioOutputDevice` which is `string | null` and is covered by `AudioInputDevice | VideoInputDevice | undefined`.

 So it's for readability.